content/en/integrations/rsyslog.md

```diff
@@ -65,14 +63,15 @@ Configure Rsyslog to gather logs from your host, containers, & services.
     * Modify your `/etc/rsyslog.d/datadog.conf` to end with the following content:
 
         ```
-        #Define the destination for the logs
+        ## Define the destination for the logs
```

Comment:
Any reason for the double escape?

Reply:
To be consistent with the comment policy of the integrations configuration files. Not a biggy though i could put back the former one `#` if you prefer